sh/tests/grades_test.sh

```diff
@@ -19,6 +19,12 @@ $input
 EOF
     )
     diff <(echo "$submitted") <(echo "$expected")
+
+    if [ $? != 0 ]; then
+        exit 1
+    fi
+    echo $submitted
+    echo $expected
```

Comment:
Those two `echo` could be removed, I guess they are a reminiscence from an earlier testing stage.

Reply:
done